app/dashboards/external_link_dashboard.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require "administrate/base_dashboard"
+
+class ExternalLinkDashboard < Administrate::BaseDashboard
+  # ATTRIBUTE_TYPES
+  # a hash that describes the type of each of the model's fields.
+  #
+  # Each different type represents an Administrate::Field object,
+  # which determines how the attribute is displayed
+  # on pages throughout the dashboard.
+  ATTRIBUTE_TYPES = {
+    versions: Field::HasMany.with_options(class_name: "PaperTrail::Version"),
+    id: Field::Number,
+    title: Field::String,
+    link: Field::String,
+    created_at: Field::DateTime,
+    updated_at: Field::DateTime,
+  }.freeze
+
+  # COLLECTION_ATTRIBUTES
+  # an array of attributes that will be displayed on the model's index page.
+  #
+  # By default, it's limited to four items to reduce clutter on index pages.
+  # Feel free to add, remove, or rearrange items.
+  COLLECTION_ATTRIBUTES = [
+    :id,
+    :title,
+    :link,
+  ].freeze
+
+  # SHOW_PAGE_ATTRIBUTES
+  # an array of attributes that will be displayed on the model's show page.
+  SHOW_PAGE_ATTRIBUTES = [
+    :versions,
```

Comment:
We don't seem to be doing this in any other dashbaord, so I think this is out of place.

Reply:
- Removed.
 -  Also changed the class the dashboard inherits from.

Reply:
@skng5 I still see this line.

Reply:
@skng5 I still see this line

Reply:
Which would now cause an error because the attribute is not defined for the dashboard.